tokio/src/io/util/empty.rs

```diff
@@ -73,6 +75,20 @@ impl fmt::Debug for Empty {
     }
 }
 
+cfg_coop! {
+    fn poll_proceed_and_make_progress(cx: &mut Context<'_>) -> Poll<io::Result<()>> {
```

Comment:
Does it break if you do this?
```suggestion
    fn poll_proceed_and_make_progress(cx: &mut Context<'_>) -> Poll<()> {
```

Reply:
Probably yes, let me try

Reply:
It did indeed work